lib/main_containers.pm

```diff
@@ -87,18 +87,18 @@ sub load_host_tests_podman {
         load_container_engine_test('podman');
         load_image_test('podman');
         load_3rd_party_image_test('podman');
-        loadtest 'containers/podman_firewall';
+        loadtest 'containers/podman_firewall' unless is_openstack;
         loadtest 'containers/buildah' unless is_sle_micro;
         # https://github.com/containers/podman/issues/5732#issuecomment-610222293
-        loadtest 'containers/rootless_podman' unless (is_sle('=15-sp1'));
+        loadtest 'containers/rootless_podman' unless (is_sle('=15-sp1') || is_openstack);
```

Comment:
I guess it's challenging, as it switches root/user consoles :)

Reply:
Oh, this should work by now. Let me check again! Thanks!

Reply:
It is not executed in [sle15sp4](https://openqa.suse.de/tests/8343826) as well. Maybe a problem with reboots ? http://kepler.suse.cz/tests/14643#step/rootless_podman/19

Reply:
Decided to not run it, same as for PC tests.